testgem.rb

```diff
@@ -0,0 +1,7 @@
+require 'verifyEmail'
+def mthd
+	puts 'Enter valid mail '
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.